### DISCUSSION TITLE
Is there anyone in this project willing to be paid to augment Attendize, or recommend a contractor who has worked with it before?

### DISCUSSION THREAD

**Discussion by CiaranTuvens** [Attendize/Attendize#1080] [Q&A]
Considering using Attendize for our small festival community, but having explored it extensively I have a few augmentations that we would need.

I would be willing to pay a contractor, on top of the SAAS license fee, for someone reliable. The tweaks are small, and honestly they should probably be added to the main branch. I just don't want to get taken for a ride by someone on Upwork who claims to be a Laravel developer.

Any ideas?

**Comment by gainer-twister** [comment]
Hi,
We have been using Softteco. Their developers are in Eastern Europe -
Poland mostly.
Send an email to ***@***.*** and ***@***.*** to
schedule a meeting.
They are very responsive and have full stack support for all related
efforts such as devops, hosting, javascript, etc.
We have a very customized version of Attendize to support the sport of
springboard diving so they know it inside and out.
Rates are typically between $30-40 per hour.

**Reply by CiaranTuvens** [reply]
Thanks a lot!

Those emails are displaying as ***@***.*** and ***@***.***

I guess I will just use the contact form on their website.

**Reply by aethatsme** [reply]
what are you looking to do? maybe I can help

**Comment by CiaranTuvens** [comment]
@aethatsme 

Thanks for your comment.

I would need to do some small augmentations like a support user with limited permissions, some changes to the Offline payment option where there is only-offline, only-offline, or either options. A couple of other details.

Depending upon how the project has developed since the time I installed it and was playing around pre-covid I may need a couple of other things. At the time I remember I needed some bug fixes, and to remove the PayPal payment option because it looked insecure. This could be done by upgrading my current installation, which was only used for a couple of events, or it can be done through a fresh install.

**Reply by aethatsme** [reply]
@CiaranTuvens 

The user permissions have been address on one of the branches I believe. Regarding the offline payments, I recommend not using this. I've used this in a live setting and it was a headache afterwards. Not all payments were authorized and ended up losing money (bad credit cards etc...)

Paypal option is the most secured for the customer because it takes you to the paypal site to process the payment. The downside is that customers leave your site. The paypal option can easily be removed. I personally use the stripe option.

I recommend doing a fresh install and go from there. Many of the features you are looking for have been addressed in the latest install.

**Reply by aethatsme** [reply]
If you're still needing help, don't hesitate to reach out.

**Reply by CiaranTuvens** [reply]
Can you give your email or a contact, or email me on ciaran @ Tuvens .com

If you can install Attendize for me I'm happy to pay. I'm not a developer and lost touch with the guy helping me last time.

**Reply by aethatsme** [reply]
Email sent.

**Comment by bastelix** [comment]
Hello community,

we are a small development agency from Germany that has started in 2020 hosting and developing attendize for a customer. So we have noticed attendize is a very good system for planning company wide events and the customer is using it for more events than planned before. We have further developed and integrated lot of stuff and using attendize with autodeploy to an docker container. We can develop each functionality really fast if needed.

So we have built an complete PDF Templates system for a customer to add your own PDF templates with the use of form variables. Also there is a strong easy role management implemented and some easy helper like upload and show own documents or own background image for the frontend. At the moment we have some free capacity for new longterm customer. If you are interessted in hosting and further development of attendize in an agile way, lets talk.

If someone is interessted so you can ping me at [eventbuchung.online](https://eventbuchung.online) or [eventbooking.online](https://eventbooking.online) , so for this domain u can get your new sub domain and there own event management system;)

regards
Bastelix
calHelp.de
